src/components/newComponent/pos/PaymentModal.tsx

```
@@ -0,0 +1,280 @@
+                );
+              })}
+            </div>
+
+            {selectedMethod && (
+              <div className="space-y-3">
+                <div className="grid grid-cols-2 gap-3">
+                  <div>
+                    <label className="block text-sm font-medium text-gray-700 mb-1">
+                      Montant (XAF)
+                    </label>
+                    <Input
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)